### ISSUE TITLE
getblocktemplate stops including basic_zelnode_address and bamf_zelnode_address 

### ISSUE THREAD

**Issue by slavikus** [zelcash/zelcash#74]
Every once in a while, a running zelcashd stops returning certain payment-related fields in the `getblocktemplate` response until restarted:

```
{
  "capabilities": [
    "proposal"
  ],
  "version": 4,
  "previousblockhash": "000000325d9546f85a03992ab0bce6dd7ffa45d6b9445ac068cbc4e0040f38ab",
  "finalsaplingroothash": "1df2e5e96c4baba421eb14e8a09e1c5cefbcba68dea6c9310435adab99d0e0a2",
  "transactions": [
  ],
  "coinbasetxn": {
    "data": "0400008085202f89010000000000000000000000000000000000000000000000000000000000000000ffffffff0503c8370600ffffffff02a0b83046030000001976a91404e2699cec5f44280540fb752c7660aa3ba857cc88ac601de137000000001976a914e55c7c067e5e34fb6f749743e5843301f9ffdea988ac00000000000000000000000000000000000000",
    "hash": "0dde52dbf4df6498608000138551da8c63d28f77878ea2776a1e6010ae56633f",
    "depends": [
    ],
    "fee": 0,
    "sigops": 2,
    "required": true
  },
  "longpollid": "000000325d9546f85a03992ab0bce6dd7ffa45d6b9445ac068cbc4e0040f38ab142834",
  "target": "00000042ee1d0000000000000000000000000000000000000000000000000000",
  "mintime": 1566420859,
  "mutable": [
    "time",
    "transactions",
    "prevblock"
  ],
  "noncerange": "00000000ffffffff",
  "sigoplimit": 20000,
  "sizelimit": 2000000,
  "curtime": 1566421581,
  "bits": "1d42ee1d",
  "height": 407496,
  "miner_reward": 14062500000,
  "super_zelnode_address": "t1enMYCUYBJxamSqvCYLhvoUv4NizCZvfFT",
  "super_zelnode_payout": 937500000
}
```

As you can see, `basic_zelnode_address`, `basic_zelnode_payout`, `bamf_zelnode_address`, `bamf_zelnode_payout` fields are missing.

Restarting zelcashd returns the expected behavior.

**Comment by milesmanley** [comment]
Hey, thanks for the report. We're aware and v3.3.1 should resolve this - currently in review before release later this week we hope.

**Comment by milesmanley** [comment]
Will leave open for other reports of similar issue - post v3.3.1 release so we can see how affective the release was in combating this. @blondfrogs

**Comment by milesmanley** [comment]
Resolved by v3.3.1 - please feel free to open another issue if this is still persistent.
